Library/Homebrew/cask/caskroom.rb

```diff
@@ -25,7 +25,7 @@ def self.paths
     # Return all tokens for installed casks.
     sig { returns(T::Array[String]) }
     def self.tokens
-      paths.map(&:basename).map(&:to_s)
+      paths.map { |path| path.basename.to_s }
```

Comment:
This feels a bit weird and unlike traditional Ruby patterns. Which cop suggested it?

Reply:
`Performance/MapMethodChain`. Two maps is slower than one.

Reply:
@Bo98 Yeh, feels like a micro-optimisation to me and not sure I agree with it really but: not enough to object.

Reply:
yeah, it might matter over the Formula API hash where we've seen the smallest of things become visible, but probably not for any of these.

Reply:
This could also be `paths.map { _1.basename.to_s }`, which is the most readable option imo (it avoids the duplication of `map` or `path` in the other styles).

Reply:
Personally I hate the `_1` variables 😅